canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -374,6 +385,25 @@ class ToolTip extends React.Component {
 			tipClass += " " + this.props.className;
 		}
 
+		let link = null;
+		if (this.state.isTooltipVisible && this.props.tooltipLinkHandler && this.props.link) {
+			const linkInformation = this.props.tooltipLinkHandler(
+				this.props.link.propertyId,
```

Comment:
Instead of passing these back as 3 different parameters maybe we just pass back this.props.link then the tooltip wouldn't have to know about propertyId.